opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -44,6 +44,33 @@ message InstrumentationLibraryMetrics {
   repeated Metric metrics = 2;
 }
 
+// A representation of raw measurements, which can have statistical meaning based
+// on how the measurement was sampled, as well as a span ID and trace ID of the
+// active span when the measurement was recorded
+message RawValue {
+  // The set of labels that were dropped by the aggregator, but recorded
+  // alongside the original measurement. Only labels that were dropped by the aggregator should be included
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
+
+  // Exact time that the measurement was recorded
+  fixed64 time_unix_nano = 2;
```

Comment:
// Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.

Reply:
done